### PR TITLE
[airquality] Update README.md

### PR DESCRIPTION
**Pull request by glebster-d**
# Fixing wrong channels in items

[Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
660 York Street, Suite 102,
San Francisco, CA 94110 USA

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Gleb Dyskin <47859640+glebster-d@users.noreply.github.com>

### CODE REVIEW

**Comment by jlaur** [comment]
@glebster-d, can you fix sign-off? See https://www.openhab.org/docs/developer/contributing.html#contributing-to-the-documentation.

File: bundles/org.openhab.binding.airquality/README.md [hunk 1]
```diff
@@ -127,19 +127,19 @@ airquality.items:
...
-DateTime Aqi_ObservationTime "Time of observation [%1$tH:%1$tM]" <clock> (AirQuality) { channel="airquality:station:local:aqi#timestamp" }
+DateTime Aqi_ObservationTime "Time of observation [%1$tH:%1$tM]" <clock> (AirQuality) { channel="airquality:station:main:home:aqi#timestamp" }
 
-Number:Temperature  Aqi_Temperature     "Temperature" <temperature> (AirQuality) { channel="airquality:station:local:weather#temperature" }
-Number:Pressure     Aqi_Pressure        "Pressure" <pressure> (AirQuality) { channel="airquality:station:local:weather#pressure" }
-Number:Dimensionless Aqi_Humidity        "Humidity" <humidity> (AirQuality) { channel="airquality:station:localweather#humidity" }
+Number:Temperature  Aqi_Temperature     "Temperature" <temperature> (AirQuality) { channel="airquality:station:main:home:weather#temperature" }
+Number:Pressure     Aqi_Pressure        "Pressure" <pressure> (AirQuality) { channel="airquality:station:main:home:weather#pressure" }
+Number:Dimensionless Aqi_Humidity        "Humidity" <humidity> (AirQuality) { channel="airquality:station:main:home:weather#humidity" }
 ```
```
Comment (glebster-d) [inline bundles/org.openhab.binding.airquality/README.md:144]: Signed-off-by: Gleb Dyskin 47859640+glebster-d@users.noreply.github.com

Comment (glebster-d) [inline bundles/org.openhab.binding.airquality/README.md:144]: Signed-off-by: Gleb Dyskin 47859640+glebster-d@users.noreply.github.com

**Comment by lolodomo** [comment]
@jlaur : for small changes and only in documentation, we accept changes done directly in GitHub (you identify them as branch is named patch-1) with a failing DCO.

**Comment by jlaur** [comment]
> @jlaur : for small changes and only in documentation, we accept changes done directly in GitHub (you identify them as branch is named patch-1) with a failing DCO. 

Okay, documentation is wrong then?

"The sign off rules described above do apply to documentation contributions as well."

**Comment by lolodomo** [comment]
No, documentation is correct... but when someone changed just few words in README, it arrived in the past we bypassed the general use and merged even if DCO was wrong.

**Comment by glebster-d** [comment]
There were a couple of mistakes in documentation, so if you copy the code sample as is it won't work. I tried it by myself and after small fixes, everything is fine

**Review by jlaur — APPROVED** [review]
Looks good, thanks.

**Pull request merged**